app/src/main/res/values/styles_congress.xml

```diff
@@ -81,10 +83,6 @@
         <item name="android:minHeight">?attr/actionBarSize</item>
     </style>
 
-    <style name="PopupMenuOverflowStyle" parent="@style/Widget.AppCompat.Light.PopupMenu.Overflow">
```

Comment:
Now the style isn't available at all for pre-API21 devices. This is what it looks like on an API 16 emulator.

![eventfahrplan_menu_without_background](https://user-images.githubusercontent.com/218061/82352562-757b9e00-99fe-11ea-96f6-279d2dc07837.png)

You need to keep the definition in this file.

Reply:
OMG! Fixed in 4b5469fce43e76a0c7be71f3395873c34e6991d2. Successfully verified on Android 5.0.2 emulator and Android 10 device.